### PR TITLE
Contributing guide

### LINKED ISSUE #69: Document contributor workflow
We need to improve our documentation around contributing to the project, both in terms of committing code and participating in the analysis/discussion.

The goal is to create a top-level "Contribute" navigation item on the website that links to a section in the [About](httparchive.org/about) page with the following info:

- We use **Slack** ([bit.ly/http-archive-slack](https://bit.ly/http-archive-slack)) for on-topic banter and administrative conversations. The #general channel is open to the public.
- We use the **forum** ([discuss.httparchive.org](https://discuss.httparchive.org)) for sharing queries and analysis related to the BigQuery datasets.
- We use **GitHub** ([HTTPArchive/httparchive.org](https://github.com/HTTPArchive/httparchive.org)) for bug reports and feature requests related to the system infrastructure. We tag issues as [Good first issue](https://github.com/issues?q=is%3Aissue+user%3AHTTPArchive+label%3A%22Good+first+issue%22+is%3Aopen) if it would be a good place for new contributors to start.
- We use **Hangouts** (TBD) for monthly meetings to discuss administrative tasks. This meeting is open to the public and meeting notes will be made available.
- We use **Twitter** ([@HTTPArchive](twitter.com/HTTPArchive)) for promoting topics of general interest to the project.

If a person wants to get more involved with the project, they should understand when to use which channel and how to join.

### PR DESCRIPTION
**Pull request by rviscomi**
Closes #69 

New section on the About page documenting all the ways to contribute to the project.

image

Please also feel free to suggest any wordsmithing or UI improvements!

### CODE REVIEW

File: templates/about.html [hunk 1]
```diff
@@ -18,6 +18,36 @@ <h2>
...
+			<li>
+				<a href="https://discuss.httparchive.org">Post on the HTTP Archive forum</a> to share interesting analyses, get SQL help, and follow official product announcements.
+			</li>
+
+			<li>
+				<a href="https://github.com/HTTPArchive/httparchive.org">Contribute to the HTTP Archive on GitHub</a> to file a bug or feature request, submit a code change, or learn more about how it works under the hood.
+			</li>
+
+			<li>
+				<a href="https://docs.google.com/document/d/1D4k77yFVbk1e-l28sTC2fniqYQIyyt49OLPOuXTgx8g/edit?usp=sharing">Join the monthly maintenance Hangout</a> to get an update on administrative tasks. This meeting is open to the community and notes are made available.
```
Comment (igrigorik) [inline templates/about.html]: Aside: this hangout link triggers a "ask to join prompt", which means creator must explicitly accept the request (and be on the call). Is there a way we could have it open by default?

File: templates/about.html [hunk 2]
```diff
@@ -18,6 +18,36 @@ <h2>
 		</p>
 	</section>
 
+	<section id="contribute" class="container">
+		<h2>Contribute</h2>
+
+		<p>
+			The HTTP Archive is an open source project maintained by a core group of developers and contributors from the community. Here's how you can become more involved with the project:
```
Comment (igrigorik) [inline templates/about.html]: I propose we..

- (a) link "core group of developers and contributors" to https://httparchive.org/faq#who-maintains-the-http-archive
- (b) update said page to include Paul + restructure that section a little bit by clarifying honorary members vs current maintainers. Something like

> Current core maintainers are X, Y, Z, D. 
> 
> Many people have contributed(link) and helped make HTTP Archive successful over the years. Special thanks to Steve Souders, who started the project in 2010, Pat Meenan who built the WebPageTest infrastructure powering HTTP Archive, and Guy Leech and Stephen Hay for design help along the way.

**Comment by paulcalvano** [comment]
This looks good to me.  

Are we making the monthly meeting notes available somewhere, and if so is there a link that we should include on this page?

File: templates/about.html (hunk 2 above)
Comment (rviscomi) [reply templates/about.html]: Done

image

image

File: templates/about.html (hunk 1 above)
Comment (rviscomi) [reply templates/about.html]: Hmm I looked into this and found https://support.google.com/a/answer/6138421?hl=en&ref_topic=6052417 but it seems a hangout within an organization that is open to the public must have the "knock to enter" function :-/.. I did set that config to enable all HA org hangouts to be public by default.

**Comment by rviscomi** [comment]
@paulcalvano yes, meeting notes will be made public. I made the Hangout link to go the notes doc, which in turn includes a link to the Hangout.

File: templates/about.html (hunk 1 above)
Comment (igrigorik) [reply templates/about.html]: What if we created a hangout link from one of our personal accounts? Would that remove the need for knock to enter? Or, are there other gotchas associated with that route?

Comment (rviscomi) [reply templates/about.html]: Per https://support.google.com/meet/answer/7308559?hl=en "You need a G Suite account to create meetings" 😞

But this limitation seems to apply to Meet conferencing, not legacy Hangouts. So if I create a personal Calendar event with a Hangout attached, I can join from other accounts via permalink without knocking.

Updated the VC link in the doc. Let's see how well it works next time we meet 😄

**Comment by rviscomi** [comment]
@igrigorik LGTY?

File: docs/faq.md [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 ### How does the HTTP Archive decide which URLs to test?
 
-HTTP Archive crawls 500,000 URLs on both desktop and mobile twice a month. The URLs come from the most popular 500,000 sites in the [Alexa Top 1,000,000](http://www.alexa.com/topsites) list.
+The HTTP Archive crawls [~1.3 million URLs](https://httparchive.org/reports/state-of-the-web#numUrls) on both desktop and mobile twice a month. The URLs come from the Chrome User Experience Report, a monthly dataset of real user performance data of the most popular websites. Only the URLs whose domain is in the [Alexa Top 1,000,000](http://www.alexa.com/topsites) ranked list are included.
```
Comment (igrigorik) [inline docs/faq.md]: "1M+ URLs" to future proof the text a bit?

File: templates/about.html (hunk 1 above)
Comment (igrigorik) [reply templates/about.html]: /cc @yoavweiss fyi. we have the same challenge with the webperf WG meetup, where we need to manually accept folks from a chromium.org account. Might be worth exploring similar solution there..

File: docs/faq.md (hunk 3 above)
Comment (rviscomi) [reply docs/faq.md]: Done

**Pull request merged**